docs/index.rst

```diff
@@ -30,7 +30,7 @@ Links
 ^^^^^
 
 * `Online documentation <https://bodhi.fedoraproject.org/docs/>`_
-*  IRC: #bodhi on Freenode
+* `IRC: #bodhi on Freenode <https://webchat.freenode.net/>`_
```

Comment:
Since the link doesn't actually take the person to the #bodhi channel but just to Freenode, can we adjust it so the link is only on the word "Freenode"? Like this:

```
* IRC: #bodhi on `Freenode <https://webchat.freenode.net/>`_
```